src/sonic-config-engine/minigraph.py

```diff
@@ -739,6 +742,21 @@ def parse_dpg(dpg, hname):
                         if panel_port not in intfs_inpc and panel_port not in acl_intfs:
                             acl_intfs.append(panel_port)
                     break
+            if aclintf.find(str(QName(ns, "Type"))) is not None and aclintf.find(str(QName(ns, "Type"))).text.upper() == "BMCDATA":
+                if 'v6' in aclname.lower():
+                    is_bmc_data_v6 = True
+                    acl_table_types['BMCDATAV6'] = {
+                        "ACTIONS": "PACKET_ACTION,COUNTER",
+                        "BIND_POINTS": "PORT",
+                        "MATCHES": "SRC_IPV6,DST_IPV6,ETHER_TYPE,IP_TYPE,IP_PROTOCOL,IN_PORTS,TCP_FLAGS",
+                    }
+                else:
+                    is_bmc_data = True
+                    acl_table_types['BMCDATA'] = {
+                        "ACTIONS": "PACKET_ACTION,COUNTER",
+                        "BIND_POINTS": "PORT",
+                        "MATCHES": "SRC_IP,DST_IP,ETHER_TYPE,IP_TYPE,IP_PROTOCOL,IN_PORTS,TCP_FLAGS",
+                    }
```

Comment:
How about defining a global dict, which includes the `ACTIONS`, `BIND_POINTS` and `MATCHES`?  We can separate the data and logic in this way, and it would be easier to add other custom types.

Reply:
Thanks @bingwang-ms! Per your suggestion, updated the code.